lisa/operating_system.py

```diff
@@ -198,19 +236,46 @@ def name_pattern(cls) -> Pattern[str]:
     def _initialize_package_installation(self) -> None:
         self._node.execute("apt-get update", sudo=True)
 
-    def _install_packages(self, packages: Union[List[str]]) -> None:
+    def _install_packages(
+        self, packages: Union[List[str]], signed: bool = True
+    ) -> ExecutableResult:
         command = (
             f"DEBIAN_FRONTEND=noninteractive "
             f"apt-get -y install {' '.join(packages)}"
         )
-        self._node.execute(command, sudo=True)
+        if not signed:
+            command = command.__add__(" --allow-unauthenticated")
+
+        return self._node.execute(command, sudo=True)
+
+    def _get_os_version(self) -> OsVersion:
+        os_version = OsVersion("")
+        cmd_result = self._node.execute(cmd="which lsb_release", no_error_log=True)
+        if cmd_result.exit_code != 0 and cmd_result.stdout != "":
+            os_version.vendor = self._node.execute(
+                cmd="lsb_release -i -s", no_error_log=True
+            ).stdout
+            os_version.release = self._node.execute(
+                cmd="lsb_release -r -s", no_error_log=True
+            ).stdout
+            os_version.release = self._node.execute(
+                cmd="lsb_release -c -s", no_error_log=True
+            ).stdout
+            if os_version.vendor in ["Debian", "Ubuntu", "LinuxMint"]:
```

Comment:
What the package info is for? Can it just be set to "deb" here?

Reply:
> What the package info is for? Can it just be set to "deb" here?

It wouldn't be "deb" if the distro vendor is not among those in the list, for example for SUSE, package wouldn't be "deb"

Reply:
suse shouldn't get into this class, this class is for debian and its derives.